tests/pages/firefox_dnt.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import BasePage
+
+
+class FirefoxDNTPage(BasePage):
+
+    _url = '{base_url}/{locale}/firefox/dnt'
+
+    _dnt_status = (By.CSS_SELECTOR, '#dnt-status > h5')
+    _faq_1_header = (By.ID, 'faq-1-header')
+    _faq_1_panel = (By.ID, 'faq-1-panel')
+
+    def dnt_status_is_visible(self):
+        el = self.selenium.find_element(*self._dnt_status)
+        return el.is_displayed()
+
+    def click_faq_1_header(self):
+        self.selenium.find_element(*self._faq_1_header).click()
+        Wait(self.selenium, self.timeout).until(
+            expected.visibility_of_element_located(self._faq_1_panel))
+
+    def faq_1_panel_is_visible(self):
+        el = self.selenium.find_element(*self._faq_1_panel)
+        return el.is_displayed()
```

Comment:
If you create a `PageRegion` for frequently asked questions, such as:

```python
class FrequentlyAskedQuestion(PageRegion):

    _question_locator = (By.TAG_NAME, 'h3')
    _answer_locator = (By.TAG_NAME, 'div')

    def show_answer(self):
        assert not self.is_answer_displayed, 'Answer is already displayed'
        self.root.find_element(*self._question_locator).click()
        Wait(self.selenium, self.timeout).until(
            expected.visibility_of_element_located(self._answer_locator))

    @property
    def is_answer_displayed(self):
        return self.root.find_element(*self._answer_locator).is_displayed()
```

Then you could create and return this page region for each question on the page. In the main page object you could have something like:

```python
_faqs_locator = (By.CSS_SELECTOR, '#faq > section')

@property
def frequently_asked_questions(self):
    return [FrequentlyAskedQuestion(self.selenium, root=el) for el in \
               self.selenium.find_elements(*self.faqs_locator)]
```

Then in the test you could check the first item like so:

```python
question = page.frequently_asked_questions[0]
question.show_answer()
assert question.is_answer_displayed
```

Or you could easily check them all:

```python
for question in page.frequently_asked_questions:
    question.show_answer()
    assert question.is_answer_displayed
```

I'd also suggest having a `hide_question` to test that functionality. Wherever you assert something is visible, I recommend using the same property to check the opposite - if at all possible. This reduced the chance of false positive.

Note that my code samples are untested. :smile: